fields/types/sortableRelateds/SortableRelateds.js

```diff
@@ -272,11 +249,19 @@ module.exports = Field.create({
   renderSelect(noedit) {
     return (
       <div>
-        <Select
+        <Select.Async
+          multi
+          simpleValue
+          labelKey="slug"
+          valueKey="id"
+          placeholder="Select..."
           disabled={noedit}
-          options={this.state.options}
-          onChange={this.onOptionChange}
-          value={this.state.selectedOption}
+          clearable={false}
+          backspaceRemoves={false}
+          value={this.state.value}
+          loadOptions={this.loadOptions}
+          onChange={this.onValueChange}
+          valueComponent={(props) => { return null; }}
```

Comment:
Question: why do we need to return custom null component for `valueComponent` ?

Reply:
because we are trying to customized the multi select of `react-select` and don't want to show the selected options like default:

<img width="924" alt="截圖 2022-05-12 下午12 12 42" src="https://user-images.githubusercontent.com/25971696/167990828-d024ba4f-eecd-4fd2-8576-6a30c5e55f23.png">

so the simplest way I found is to use `valueComponent` props and let it show nothing.